File: java/listview-java/app/src/main/java/com/bmarques/listview/model/Order.java
package com.bmarques.listview.model;

import java.io.Serializable;

public class Order implements Serializable {

    private Integer id;
    private String client;
    private Double value;

    public Order(String client,
                 Double value) {
        this.client = client;
        this.value = value;
    }

    public Order() {
    }

    public String getClient() {
        return client;
    }

    public void setClient(String client) {
        this.client = client;
    }

    public Double getValue() {
        return value;
    }

    public void setValue(Double value) {
        this.value = value;
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    @Override
    public String toString() {
        return client;
    }
}

File: kotlin/ListViewKotlin/app/src/main/java/com/bmarques/listviewkotlin/model/Order.kt
package com.bmarques.listviewkotlin.model

import java.io.Serializable

class Order(val client: String, val value: Double) : Serializable {

    var id: Int = 0
        private set

    fun updateIds(newId: Int) {
        id = newId
    }

    override fun toString(): String {
        return client
    }

}
